packages/replies/src/replies.tsx

```diff
@@ -36,10 +32,8 @@ export default function Replies({
   size?: number
   onClickCapture?: (event: MouseEvent<HTMLDivElement>) => void
 }) {
-  const [totalRepliesCount, setTotalRepliesCount] = useState<
-    number | undefined
-  >(undefined)
   const [replies, setReplies] = useState<Reply[]>([])
+  const [haveNextReplies, setHaveNextRelies] = useState(false)
```

Comment:
네이밍은 이렇게 하는 게 좋겠어요. 그리고 조만간 서버사이드에 작업해서 이 정보 같이 줄 수 있게 해야겠네요.

```suggestion
  const [hasNextPage, hasNextPage] = useState(false)
```